test/index.html

```diff
@@ -3,15 +3,10 @@
 <head>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width">
-  <title>QUnit Example</title>
-  <link rel="stylesheet" href="https://code.jquery.com/qunit/qunit-2.0.0.css">
+  <title>Browser-passworder tests</title>
 </head>
 <body>
-  <div id="qunit"></div>
-  <div id="qunit-fixture"></div>
-  <script src="https://code.jquery.com/qunit/qunit-2.0.0.js"></script>
-  <script src="./jquery-3.1.0.min.js"></script>
+  This page is used to run browser-passworder unit tests. This page is not intended to be used directly.
```

Comment:
AFAIR you don't need a page to run tests with playwright

Reply:
Resolving this thread so we can discuss it in the other one; I think the two comments are related enough.